core/templates/dev/head/editor/exploration_history.html

```diff
@@ -18,24 +18,23 @@
     <h3>List of Changes</h3>
     <div ng-repeat="versionNumber in snapshotOrderArray" class="row">
```

Comment:
Should we still be calling this 'versionNumber'? It seems like that would be a number, but now it's an object.

Maybe snapshotMetadata or something like that?